tests/TestLinchPinCredentials.py

```diff
@@ -0,0 +1,152 @@
+import os
```

Comment:
https://www.python.org/dev/peps/pep-0008/#package-and-module-names implies that the filenames here need to be test_linchpin_credentials.py. Can you fix this please?